<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistrationWidget</class>
 <widget class="QWidget" name="RegistrationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableView" name="teamView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>261</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>41</width>
     <height>113</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="addTeamButton">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/icons/icons/add.png</normaloff>:/icons/icons/add.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="deleteTeamButton">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/icons/icons/delete.png</normaloff>:/icons/icons/delete.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="editTeamButton">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/icons/icons/edit.png</normaloff>:/icons/icons/edit.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
